module/VuFind/src/VuFind/Http/CachingDownloader.php

```diff
@@ -187,15 +188,16 @@ public function download(
      * Download a resource using the cache in the background,
      * including decoding for JSON.
      *
-     * @param string $url    URL
-     * @param array  $params Request parameters (e.g. additional headers)
+     * @param string    $url         URL
+     * @param array     $params      Request parameters (e.g. additional headers)
+     * @param bool|null $associative Sent to json_decode
      *
-     * @return stdClass
+     * @return stdClass|array
```

Comment:
Looks like another instance of using stdClass without a use statement (which I realize was a pre-existing problem here).

Reply:
Fixed